ClassLoader/src/main/java/company/Helper.java

```diff
@@ -0,0 +1,53 @@
+package company;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Method;
+import java.net.URL;
+import java.net.URLClassLoader;
+import java.util.Scanner;
+
+public class Helper {
+
+    Class<?> aClass = null;
+    Logger log = LogManager.getLogger(Helper.class);
+    public Helper() {
+        while (true){
+            log("Menu: \n1-start \n0-exit \n");
```

Comment:
Better to use some specific logging level like debug, info, error, etc.
please update it for all places

Reply:
I already use info level